prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -197,7 +197,7 @@ metadata:
     kyma-project.io/installation: ""
 stringData:
   kafka.brokers.hostname: ${K8S_SECRET_BROKER_HOSTNAME}
-  kafka.brokers.port: ${K8S_SECRET_BROKER_PORT}
+  kafka.brokers.port: "${K8S_SECRET_BROKER_PORT}"
```

Comment:
Should we also apply this for the rest of the stringData values?

Reply:
I will check that the overrides are still working on a fresh cluster and come back to you.